joy_teleop/test/test_key_teleop.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019 Apex.AI, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import glob
+import os
+import subprocess
+
+import ament_index_python
+
+import pytest
+
+
+testdata = glob.glob(
+    os.path.join(
+        ament_index_python.get_package_share_directory('joy_teleop'),
```

Comment:
sure you want `joy_teleop` here?
_and_ if no but it works, can we remove this? :D